client/src/pages/AboutPage/AboutPage.jsx

```diff
@@ -0,0 +1,51 @@
+import "./AboutPage.css";
+
+function AboutPage() {
+  return (
+    <div className="about-wrapper">
```

Comment:
This div seems pointless as there is nothing more than in the the "about-container" div. You can apply the css rules you set on a single level of div.

Reply:
Done